sumologic/util_test.go

```diff
@@ -0,0 +1,202 @@
+package sumologic
+
+import (
+	"testing"
+)
+
+// Based on https://github.com/hashicorp/terraform-provider-aws/blob/232c3ed9c9d18aab6f6b70672d7c46d44c75a52e/internal/verify/diff_test.go#L62.
+func TestSuppressTimeDiff(t *testing.T) {
```

Comment:
I added a unit test here.
I am not sure we can create an acceptance test for the case where we want to assert that the plan was empty. If you know how to do it, please let me know, I will write such acceptance test.